dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -475,6 +475,84 @@ def wavepostsbs(self):
 
         return task
 
+    def wavestat(self):
+        deps = []
+        for member in range(0, self.nmem + 1):
+            task = f'gefs_wave_post_grid_mem{member:03d}_#fhr_label#'
+            dep_dict = {'type': 'task', 'name': task}
+            deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
+
+        fhrs = self._get_forecast_hours('gefs', self._configs['wavepostsbs'], 'wave')
+
+        # When using replay, output does not start until hour 3
+        is_replay = self._configs['wavepostsbs']['REPLAY_ICS']
+        if is_replay:
+            fhrs = [fhr for fhr in fhrs if fhr not in [0, 1, 2]]
+
+        max_tasks = self._configs['wavepostsbs']['MAX_TASKS']
+        fhr_var_dict = self.get_grouped_fhr_dict(fhrs=fhrs, ngroups=max_tasks)
+
+        wavestat_envars = self.envars.copy()
+        postenvar_dict = {'FHR_LIST': '#fhr_list#'}
+        for key, value in postenvar_dict.items():
+            wavestat_envars.append(rocoto.create_envar(name=key, value=str(value)))
+
+        resources = self.get_resource('waveinit')
```

Comment:
```suggestion
        resources = self.get_resource('wavestat')
```
Should be `wavestat`